src/site/content/en/blog/model-viewer/index.md

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
@jpmedley & @rkochman are you expecting this to be the authoritative post on model-viewer for the foreseeable future?

I'm asking because you're currently using the authoritative URL https://web.dev/model-viewer

Another option would be https://web.dev/introducing-model-viewer if you wanted to save https://web.dev/model-viewer for something else in the future

Reply:
This is pulling from another site and it works.